GettingStarted.md

```diff
@@ -57,4 +61,22 @@ Building...
 dotnet dev-certs https --trust
 ```
 
-12. To confirm the server is running properly, visit <https://127.0.0.1:5001/swagger/v1/swagger.json>, where you should get a JSON file and see `StatusCode: 200` in the server logs. Authentication and authorization are disabled in development mode. You can view the [REST API documentation](https://nightingaleproject.github.io/Reference-NCHS-API/#/) on GitHub Pages, and use the [Canary Testing Framework](https://canary.ctex-inc.com/) to generate sample FHIR messages. You can also use the [Postman Collections](https://github.com/nightingaleproject/Reference-NCHS-API/tree/766-getting-started-guide/examples) provided, but disregard authentication and use `https://127.0.0.1:5001/` as the base URL.
+12. To confirm the server is running properly, visit <https://127.0.0.1:5001/swagger/v1/swagger.json>, where you should get a JSON file and see `StatusCode: 200` in the server logs. Authentication and authorization are disabled in development mode. You can view the [REST API documentation](https://nightingaleproject.github.io/Reference-NCHS-API/#/) on GitHub Pages or <http://localhost:5001/swagger/index.html>.
+
+
+## Run Messaging Tests
+
+From `messaging` directory with MSSQL database already running, run:
+
+```shell
+dotnet test
+```
+
+## Next Steps
+
+In any order:
+
+ - Use the [Canary Testing Framework](https://canary.fhir.nvss.cdc.gov/) to generate sample FHIR messages.
+ - Try the [Postman Collections](https://github.com/nightingaleproject/Reference-NCHS-API/tree/main/examples/README.md) provided, but disregard authentication and use `https://127.0.0.1:5001/` as the base URL.
+ - Launch the [StatusUI](status_api/README.md) to get a NVSS FHIR messages status dashboard.
+ - Checkout our [Testing Tools](testing_tools/README.md) for probing the NVSS FHIR API.
```

Comment:
The Testing Tools were built specifically to easily provide lots of content for the status UI to show, so I'd 

1) Move this up to the top of the list, and
2) say something like "Check out our Testing Tools to quickly populate the FHIR API database with content suitable for exercising the Status UI